docs/API/input.md

```diff
@@ -0,0 +1,49 @@
+
+<img src="https://i.imgur.com/Zm7MTQD.png" width="300"/>
+
+```js
+
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { Input } from 'react-native-elements';
+
+<Input
+  placeholder='BASIC INPUT'
+/>
+
+<Input
+  placeholder='INPUT WITH ICON'
+  icon={
```

Comment:
`icon` props was updated with the SearchBar PR and is now `leftIcon` and `rightIcon`